tapir/shifts/tests/test_shift_template_duplicate.py

```diff
@@ -75,8 +75,14 @@ def test_shiftTemplateDuplicateForm_duplicateSingleTemplate(self):
             follow=True,
         )
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(ShiftTemplate.objects.count(), 2)
+        self.assertEqual(2, ShiftTemplate.objects.count())
         for template in ShiftTemplate.objects.all():
+            self.assertEqual(
+                template.start_date.strftime("%Y%m%d"),
+                shift_template.start_date.strftime("%Y%m%d"),
+            )
+            self.assertEqual(template.start_time, shift_template.start_time)
+            self.assertEqual(template.end_time, shift_template.end_time)
             self.assertEqual(
                 template.slot_templates.name, shift_template.slot_templates.name
             )
```

Comment:
`template.slot_template` is a QuerySet, it doesn't have a `name` attribute. I'm not sure why `template.slot_templates.name` returns `None`, but that's for sure not why we want to test. 
We need to loop over all the slot templates and test all the fields that are copied in https://github.com/SuperCoopBerlin/tapir/blob/903ddb91484fe14a95528e2b2517317395f9704d/tapir/shifts/views/management.py#L248-L253